neptune/internal/artifacts/types.py

```diff
@@ -0,0 +1,78 @@
+#
```

Comment:
Same as preceding. Maybe package under `neptune/new/artifacts/` will be more suitable? or including `internal`